src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -43,9 +43,10 @@ internal class ConvertOptions
 
         [Option(
             'a',
-            "plugin-assembly-path",
+            "plugin-assembly-paths",
             Default = null,
-            HelpText = "Path to plugin assembly containing converter types.")]
-        public string PluginAssemblyPath { get; internal set; }
+            HelpText = "Comma-separated list of paths to plugin assemblies containing converter types.",
+            Separator = ',')]
+        public string[] PluginAssemblyPaths { get; internal set; }
```

Comment:
>PluginAssemblyPaths [](start = 24, length = 19)

I thought about asking for this capability but then thought, why would we need it? is it so a developer can specify a bundle of converters and then simply change the tool type and trust it will be found? I tend to think a better model is to maintain a precise coupling between the converter and its containing assembly. Tool late now, perhaps, but my opinion right now is that this is all unnecessary machinery. Do you have a different valuable scenario in mind that I'm missing?